pkg/virtctl/credentials/addkey/addkey.go

```diff
@@ -100,25 +103,31 @@ func runAddKeyCommand(clientConfig clientcmd.ClientConfig, cmdFlags *addSshKeyFl
 	}
 
 	if shouldCreateNewSecret(cmdFlags, vm) {
-		return addSecretWithSshKey(cmd, cli, cmdFlags, vm, sshKey)
+		return addSecretWithSSHKey(cmd, cli, cmdFlags, vm, sshKey)
 	}
-	return updateSecretWithSshKey(cmd, cli, cmdFlags, vm, sshKey)
+	return updateSecretWithSSHKey(cmd, cli, cmdFlags, vm, sshKey)
 }
 
-func addSecretWithSshKey(cmd *cobra.Command, cli kubecli.KubevirtClient, cmdFlags *addSshKeyFlags, vm *v1.VirtualMachine, sshKey string) (err error) {
+func addSecretWithSSHKey(cmd *cobra.Command,
+	cli kubecli.KubevirtClient,
+	cmdFlags *addSSHKeyFlags,
+	vm *v1.VirtualMachine,
+	sshKey string) error {
```

Comment:
I am unsure to which format pattern you follow when breaking the lines.

This comment is general, not only related to this incident.

I know of these format patterns:
The trivial `foo(a, b ,c)`.
The next best thing:
```
foo(
    a, b, c,
)
```

and the one-per-line one:
```
foo(
    a,
    b,
    c,
)
```

In this case I would go with the last, e.g.
```
func addSecretWithSSHKey(
	cmd *cobra.Command,
	cli kubecli.KubevirtClient,
	cmdFlags *addSSHKeyFlags,
	vm *v1.VirtualMachine,
	sshKey string,
) error {
```

As we do not have a formatter that enforces this automatically, it is not a must.
But I have hard time following the arguments when it is not well organized.